source/openpulse/setup.cfg

```diff
@@ -0,0 +1,39 @@
+[metadata]
+name = openpulse
+url = https://github.com/Qiskit/openqasm
```

Comment:
```suggestion
url = https://github.com/Qiskit/openqasm/source/openpulse
```
?

Reply:
I updated the url to `https://github.com/openqasm/openqasm/` for now. To further link into the project, github added `tree/main` in the url, such as `https://github.com/openqasm/openqasm/tree/main/source/openqasm`. We don't have a valid url for `openpulse` yet until it is merged into main.